src/hub/action_request.ts

```diff
@@ -355,6 +355,14 @@ export class ActionRequest {
       } else {
         return sanitizeFilename(`looker_file_${Date.now()}.${this.attachment.fileExtension}`)
       }
+    } else {
+      if (this.formParams.format) {
```

Comment:
Not necessary but I would recommend changing this to an `else if` and always having a return in every codepath.

```js
if (this.attachment) {
  /** Do sstuff */
}
else if (this.formParams.format) {
  /** do else if stuff */
}

return 'defaultSuggestedFilename'
```